gradle-plugin/build.gradle

```diff
@@ -13,35 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 plugins {
   id 'java-gradle-plugin'
   id 'groovy'
-  id 'maven'
+  id 'maven-publish'
```

Comment:
Switched to use maven-publish plugin (rather than maven plugin, which is outdated).